tests/test_client.py

```diff
@@ -240,7 +240,7 @@ def test_rpc_error(self):
 
         try:
             self.sg.info()
-        except api.Fault, e:
+        except api.Fault as e:
```

Comment:
Ya, I didn't recall that py24/25 don't support "as"... it's been so long. This goes into way deeper than this PR, but can we not just say "we're deprecating support for py24/25 in future versions of the API"? I'm sure any studio using those versions are very much aware that they're dead and burried--- more-so, it puts an unnecessary support burden on us in needing to maintain a version of Python that's been deprecated since before Shotgun even was a product.